src/Mod/TechDraw/Gui/QGIFace.cpp

```diff
@@ -165,16 +162,14 @@ void QGIFace::setPrettyNormal() {
 /// show the face style & colour in pre-select configuration
 void QGIFace::setPrettyPre() {
 //    Base::Console().Message("QGIF::setPrettyPre()\n");
-    m_fillStyleCurrent = Qt::SolidPattern;
-    m_brush.setTexture(QPixmap());
+    m_brush.setStyle(Qt::SolidPattern);
```

Comment:
You sure about this one?  Setting the fill style to anything other than texture fill  is supposed to clear the texture, but I have a vague memory of issues without this.   Up to date Qt might not have the issue any more.

Reply:
Hm... if the texture is not cleared, what will happen then? We don't need to clear the texture anyway if it won't be drawn?

I can put it back in, if you prefer